README.md

```diff
@@ -72,7 +73,7 @@ module "storage_buckets_iam_bindings" {
 
 ### Variables
```

Comment:
I think we could probably drop these variables? Or possibly just focus on an explanation of `Additive and Authoritative Modes` (combine with the section below).

Reply:
agreed